assets/js/main.js

```diff
@@ -121,6 +126,7 @@ $(() => {
                     .done(data => {
                         if(data.results.length > 0) {
                             const result = data.results[0];
+                            updateUnit(data);
```

Comment:
Could you move it so it isn't in the middle of two variable declarations?

Reply:
Done!